### ISSUE TITLE
[1.3.0 (27)] Asus Zenfone Go 4.5 / Android 5.1 - Black thumbnail display for motifs - Play any motif and take the app in background and again take in front end will display black thumbnail

### ISSUE THREAD

**Issue by hiteshbheda** [Lomotif/lomotif-android-issues#268]
Frequency: Every time

Action:
1. Start Lomotif application
2. Click on Start
3. Select any motif category from motif tab
4. Tap on Play button on any motif to play lomotif on screen
5. During playing motif minimize the app or take the app in background
6. Again immediately, take the app in front
7. Now check whether motif is still playing or not

Expected result: Motif should continue playing as user has not stopped it

Actual result: Instead of continue playing, on the space of motif thumbnail Black screen appears

**Please check attached video:**
[Video](https://drive.google.com/drive/u/0/folders/0B6P2U77GNA4URVU0TFJCNHp1VXM)

**Comment by xiuie** [comment]
If you click it it continue playing?

**Comment by hiteshbheda** [comment]
@xiuie Yes, when I click on it, it starts playing. But if we perform above steps then it gives black screen instead of thumbnail.

**Comment by testershub** [comment]
I can reproduce this issue for build [1.3.0(27)] on Samsung Galaxy Note 4 / 6.0.1 and Samsung Galaxy S4 / 5.0.1.

**Comment by hiteshbheda** [comment]
@xiuie I can reproduce this issue for build [1.3.0 (28) ] on Asus Zenfone Go 4.5 / Android 5.1

**Comment by hiteshbheda** [comment]
@xiuie I can reproduce this issue for build [1.3.0 (28)] on Sony xPeria Z / Android 5.1.1
